integration/admin/identify_test.go

```diff
@@ -0,0 +1,106 @@
+// +build integration
+
+package governance
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hyperledger/burrow/config"
+	"github.com/hyperledger/burrow/consensus/tendermint"
+	"github.com/hyperledger/burrow/crypto"
+	"github.com/hyperledger/burrow/rpc/rpcquery"
+
+	"github.com/hyperledger/burrow/core"
+	"github.com/hyperledger/burrow/execution/registry"
+	"github.com/hyperledger/burrow/integration"
+	"github.com/hyperledger/burrow/integration/rpctest"
+	"github.com/hyperledger/burrow/txs/payload"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIdentify(t *testing.T) {
+	accounts := integration.MakePrivateAccounts("accounts", 2)
+	kernels := make([]*core.Kernel, len(accounts))
+	configs := make([]*config.BurrowConfig, len(accounts))
+	genesisDoc := integration.TestGenesisDoc(accounts, 0)
+	var err error
+
+	for i, acc := range accounts {
+		configs[i], err = newConfig(genesisDoc, acc, accounts...)
+		require.NoError(t, err)
+		configs[i].Tendermint.IdentifyPeers = true
+	}
+
+	// start first validator
+	kernels[0], err = newKernelAndBoot(configs[0], accounts[0], accounts...)
+	require.NoError(t, err)
+	defer integration.Shutdown(kernels[0])
+
+	// identify first validator (self)
+	node, sig := nodeAndSigFromConf(t, configs[0], configs[0].Tendermint.ListenAddress())
+	identifyTx := payload.NewIdentifyTx(accounts[0].GetAddress(), node).WithSig(sig)
```

Comment:
I'm not sure `WithSig` deserves to be anything other than, at most, a test helper. It kind of obscures what we are doing here which is just setting a field mutably. I would say it is simpler and more obvious to just do that.

I do like fluent interfaces for things that are designed to be configured from code, this isn't really one of them...